src/EnergyPlus/DaylightingManager.cc

```diff
@@ -7874,8 +7862,7 @@ void DayltgInterReflectedIllum(EnergyPlusData &state,
                 if (state.dataSurface->CalcSolRefl && ObTransM(IPH, ITH) > 1.e-6) {
                     // Sun reaches ground point if vector from this point to the sun is unobstructed
                     hitObs = false;
-                    for (ObsSurfNum = 1; ObsSurfNum <= state.dataSurface->TotSurfaces; ++ObsSurfNum) {
-                        if (!state.dataSurface->Surface(ObsSurfNum).IsShadowPossibleObstruction) continue;
+                    for (int ObsSurfNum : state.dataSurface->AllShadowPossObstrSurfaceList) {
```

Comment:
The comment I made on the iteration call about function x being slower, but setting a breakpoint here never stops was only half-true. It's true that the 45ZonePTAC never hits this loop (because this loop is only when solar distribution is ...WithReflections), but this is a rather large function and it calls some functions that do contain changes from this PR. The slowness is more noise than anything. So, never mind.